[ES] Update lazer mods in `Game modifier` (3) (#15114)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Relax_(lazer)/es.md

**Nota:** Para la lista completa de todos los mods de [lazer](/wiki/Client/Release_stream/Lazer), véase [Modificadores del juego (lazer)](/wiki/Gameplay/Game_modifier_(lazer))
:::

En osu!(lazer), al completar un beatmap con Relax se envía la puntuación sin que se otorguen [puntos de rendimiento](/wiki/Performance_points). Además, a diferencia de lo que ocurre en osu!(stable), es posible fallar el beatmap durante una jugada.

### ![][osu!] osu!

En osu!, el mod **Relax** hace clic automáticamente en todos los [objetos](/wiki/Gameplay/Hit_object) de un [beatmap](/wiki/Beatmap), lo que permite al usuario jugar simplemente moviendo el cursor.

### ![][osu!taiko] osu!taiko

> En osu!taiko, el mod **Relax** le permite al usuario golpear cualquier con cualquier tecla, lo que significa que ya no es necesario distinguir qué notas son [dons](/wiki/Gameplay/Hit_object/Hit) y cuáles son [kats](/wiki/Gameplay/Hit_object/Hit).

Reviewer: ```suggestion
En osu!taiko, el mod **Relax** le permite al usuario golpear cualquier objeto con cualquier tecla, lo que significa que ya no es necesario distinguir qué notas son [dons](/wiki/Gameplay/Hit_object/Hit) y cuáles son [kats](/wiki/Gameplay/Hit_object/Hit).
```

### ![][osu!catch] osu!catch

> En osu!catch, el mod **Relax** le permite al usuario controlar al catcher con el ratón (o una tableta) en lugar del teclado. Esto también significa que el jugador ya no tiene que usar los [dash](/wiki/Gameplay/Dash).

Reviewer: ```suggestion
En osu!catch, el mod **Relax** le permite al usuario controlar al catcher con el ratón (o la tableta) en lugar del teclado. Esto también significa que el jugador ya no tiene que usar el [dash](/wiki/Gameplay/Dash).
```

**Este mod no se puede personalizar.**

## Curiosidades

> - En osu!, Relax hace clic en los objetos unos 10 ms antes de que se puedan pulsar. Como resultado, usar Relax junto con [Adaptive Speed (AS)](/wiki/Gameplay/Game_modifier/Adaptive_Speed) provocará una aceleración constante a medida que avanza el mapa, siempre y cuando el jugador apunte correctamente.

Reviewer: ```suggestion
- En osu!, Relax hace clic en los objetos unos 10 ms antes del  centro de la ventana de tiempo. Como resultado, usar Relax junto con [Adaptive Speed (AS)](/wiki/Gameplay/Game_modifier/Adaptive_Speed) provocará una aceleración constante a medida que avanza el mapa, siempre y cuando el jugador apunte correctamente.
```

> - En osu!, Relax hace clic en los objetos unos 10 ms antes de que se puedan pulsar. Como resultado, usar Relax junto con [Adaptive Speed (AS)](/wiki/Gameplay/Game_modifier/Adaptive_Speed) provocará una aceleración constante a medida que avanza el mapa, siempre y cuando el jugador apunte correctamente.

Reviewer: @AndiTafel https://github.com/ppy/osu-wiki/blob/6b11fe731ed774871a90d1e1eeca0cc413ee4acc/wiki/Gameplay/Game_modifier/Relax_(lazer)/en.md?plain=1#L61

that should be `10 ms early relative to the center of the hit window` right?

i don't understand the other part of the phrase, what does relax have to do with it?

Reviewer 2: yeah, it's an appropriate simplification
regarding the second part, it means that since relax always hits earlier, it leads to the constant speed-up when adaptive speed is activated, because it speeds the map up on early hits and slows it down on later hits

Reviewer: ah okay because adaptive speed adjusts based on the player hit timing
